chain/events/filter/index.go

```diff
@@ -118,6 +120,35 @@ func (ei *EventIndex) initStatements() (err error) {
 	return nil
 }
 
+func (ei *EventIndex) migrateToVersion3(ctx context.Context) error {
+	tx, err := ei.db.Begin()
+	if err != nil {
+		return xerrors.Errorf("begin transaction: %w", err)
+	}
+	// rollback the transaction (a no-op if the transaction was already committed)
+	defer tx.Rollback() //nolint:errcheck
+
+	if _, err = tx.ExecContext(ctx, "ALTER TABLE event_entry ADD COLUMN ordering INTEGER NOT NULL DEFAULT 0"); err != nil {
+		return xerrors.Errorf("increment _meta version: %w", err)
+	}
+
+	// Add the index on event_id and ordering fields
+	if _, err = tx.ExecContext(ctx, "CREATE INDEX IF NOT EXISTS idx_event_id_ordering ON event_entry (event_id, ordering)"); err != nil {
```

Comment:
We don't need to index on ordering, right?

Reply:
Done.